packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -130,6 +130,7 @@ export class Viewer {
 
   private isResizing?: boolean;
   private sceneViewId?: UUID.UUID;
+  private streamDisposables: Disposable[] = [];
```

Comment:
See prior comment, but I don't think this array of disposables anymore if we don't recreate `StreamApi`.